Gir.toml

```diff
@@ -579,6 +579,12 @@ status = "generate"
         [[object.function.parameter]]
         name = "argv"
         string_type = "os_string"
+    [[object.function]]
+    name = "communicate_utf8_async"
+    ignore = true
+    [[object.function]]
+    name = "communicate_utf8_async_future"
```

Comment:
You don't have to ignore that one here. It is generated as a result of the non-futures function and does not exist as such in C